hyperspy/roi.py

```diff
@@ -688,6 +703,24 @@ def _set_default_values(self, signal, axes=None):
 
     @property
     def parameters(self):
+        """dict : ROI parameters
+        Dictionary containing the parameters that define the current ROI.
+
+        For a ``Point1DROI``, the dictionary contains:
+        - ``position`` : float
+        The coordinate of the point along the axis.
+
+        Notes
+        -----
+        This property is intended for serialization and restoration of ROIs.
+
+        Examples
+        --------
+        >>> from hyperspy.roi import Point1DROI
+        >>> roi = Point1DROI(position=15)
+        >>> params = roi.parameters
+        >>> roi2 = Point1DROI(``**params``)
```

Comment:
The `` around the `**params` does not make sense here and in the other examples (it is markdown syntax for formatting code).